lint_test/prefer_match_file_name_test.dart

```diff
@@ -0,0 +1,17 @@
+// ignore_for_file: unused_element, unused_field
+
+/// Check the `prefer-match-file-name` rule
+class _AnotherPrivateClass {}
+
+class PreferMatchFileNameTest {}
```

Comment:
Should be a lint here.
```suggestion
// expect_lint: prefer-match-file-name
class PreferMatchFileNameTest {}
```

Reply:
It matches file name. 
Original rule from DCM ignores private declarations and checks only first public declaration

Reply:
Ah, my bad.